package.json

```diff
@@ -10,7 +10,7 @@
     "start": "ng serve",
     "prebuild": "ng lint && ng test --watch=false",
```

Comment:
Tests are now running twice, I'm pretty sure this wasn't the case with old config. Can be modified with --project param (for linter too I think, did not test).

Reply:
You are right, it was because of --project param (and linting was performed 2 times as well). I deleted the param and corresponding project configuration (see angular.json comment).